qiskit_machine_learning/algorithms/kernel_trainers/quantum_kernel_trainer.py

```diff
@@ -0,0 +1,171 @@
+        self._initial_point = None
+
+        # Setters
+        self.quantum_kernel = quantum_kernel
+        self.loss = loss
+        self.optimizer = optimizer
+        self.initial_point = initial_point
+
+    @property
+    def quantum_kernel(self):
+        """Returns the underlying quantum kernel."""
+        return self._quantum_kernel
+
+    @quantum_kernel.setter
+    def quantum_kernel(self, quantum_kernel: QuantumKernel) -> None:
+        """Sets the quantum kernel"""
+        self._quantum_kernel = quantum_kernel
+
+    @property
+    def loss(self):
+        """Returns the underlying loss function."""
+        return self._loss
+
+    @loss.setter
+    def loss(self, loss: Union[str, KernelLoss]) -> None:
+        """Sets the loss."""
+        if isinstance(loss, str):
+            loss = loss.lower()
+            if loss == "svc_alignment":
+                self._loss = SVCAlignment()
+            else:
+                raise ValueError(f"Unknown loss {loss}!")
+        elif callable(loss):
+            self._loss = loss
+        else:
+            raise ValueError(f"Unknown loss {loss}!")
+
+    @property
+    def optimizer(self) -> Optimizer:
+        """Returns an optimizer to be used in training."""
+        return self._optimizer
+
+    @optimizer.setter
+    def optimizer(self, optimizer: Optimizer) -> None:
+        """Sets the loss."""
+        if optimizer is None:
+            self._optimizer = SPSA(maxiter=50)
+        else:
+            self._optimizer = optimizer
+
+    @property
+    def initial_point(self) -> np.ndarray:
+        """Returns current initial point"""
+        return self._initial_point
+
+    @initial_point.setter
+    def initial_point(self, initial_point: np.ndarray) -> None:
+        """Sets the initial point"""
+        self._initial_point = initial_point
+
+    def fit_kernel(
+        self,
+        data: np.ndarray,
+        labels: np.ndarray,
+    ) -> VariationalResult:
+        """
+        Train the quantum kernel by minimizing loss over the kernel parameters.
+
+        Args:
+            data (numpy.ndarray): NxD array of training data, where N is the
+                              number of samples and D is the feature dimension
+            labels (numpy.ndarray): N x 1 array of +/-1 labels of the N training samples
+
+        Returns:
+            dict: the results of kernel training
+        """
+        # Bind inputs to objective function
+        obj_func = partial(self.loss.evaluate, kernel=self.quantum_kernel, data=data, labels=labels)
+
+        # Number of parameters to tune
+        num_params = len(self.quantum_kernel.user_parameters)
+
+        # Randomly initialize our user parameters if no initial point was passed
+        if not self.initial_point:
+            self.initial_point = algorithm_globals.random.random(num_params)
+
+        # Perform kernel done
+        ignore = self.optimizer.is_initial_point_ignored
+        print(f"Is Ignored: {ignore}")
```

Comment:
Maybe printing is useful for debugging but it should not exist in final code. If it might be helpful to have this information, such that a user can enable it to be "printed" then consider using the logger facilities.

Reply:
Oops this is a remnant of me debugging :)